solution/ui/regulations/js/src/components/ViewResourcesLink.vue

```diff
@@ -1,19 +1,36 @@
 <template>
-    <div v-if="isLink()" style="padding-left: 5px; font-size: 12px; margin-top: -10px;">
-      <a v-if="this.count !== '0'" v-on:click="clickHandler">
-        <span class="bold">View {{this.section}} resources</span> ({{ this.count }})
-      </a>
-      <div v-else class="bold disabled" >No resources for {{this.section}}.</div>
+    <div
+        v-if="isLink()"
+        class="view-resources-link"
+        style="padding-left: 5px; font-size: 12px; margin-top: -10px"
```

Comment:
The style for the v-if and the v-else are the same minus the margin-top: -10 px.  It would probably be better to add this to the same class and then add the style of just margin-top: -10px in case we want to change something.